modules/virt-subscribing-cli.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-virt-cli.adoc
+
+[id="virt-subscribing-cli_{context}"]
+= Subscribing to the {VirtProductName} catalog by using the CLI
+
+Before you install {VirtProductName}, you must subscribe to the {VirtProductName} catalog. Subscribing gives the `openshift-cnv` namespace access to the {VirtProductName} Operators.
+
+.Procedure
+
+. Create a file named `deploy.yaml` by running the following command:
++
+----
+$ touch deploy.yaml
+----
+
+. Open `deploy.yaml` in a text editor and paste the following manifest into the file:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-cnv
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kubevirt-hyperconverged-group
+  namespace: openshift-cnv
+spec:
+  targetNamespaces:
+    - openshift-cnv
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hco-operatorhub
+  namespace: openshift-cnv
+spec:
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  name: kubevirt-hyperconverged
+  startingCSV: kubevirt-hyperconverged-operator.v{HCOVersion}
+  channel: "{VirtVersion}"
+----
+
+. Save `deploy.yaml` and exit the text editor.
+
+. Create the required `Namespace`, `OperatorGroup`, and `Subscription` objects for {VirtProductName} by running the following command:
```

Comment:
It would be good to mention these required objects, and that they are defined in the yaml that we're creating, in teh intro text. I was a tad thrown reading this for the first time here.